.github/workflows/merge.yml

```diff
@@ -0,0 +1,37 @@
+
+name: Merge
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy_flatpak:
+    name: Deploy Flatpak
+    runs-on: ubuntu-latest
+
+    container:
+      image: bilelmoussaoui/flatpak-github-actions:elementary-juno
+      options: --privileged
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build
+        uses: bilelmoussaoui/flatpak-github-actions/flatpak-builder@v3
+        with:
+          bundle: org.gnome.epiphany.flatpak
+          manifest-path: org.gnome.epiphany.yml
+          run-tests: true
```

Comment:
I believe that this need to be removed here too.

Reply:
Good catch!